async function main() {
    console.log('Starting automations');
}

main()
    .then(() => console.log('Completed'))
    .catch((err) => console.log('An Error Has Occurred', err));
